subprojects/packagefiles/range-v3/meson_options.txt

```diff
@@ -0,0 +1 @@
+option('range_v3_tests', type : 'boolean', value : false)
```

Comment:
- defaulting tests off means they aren't run in the wrapdb CI
- meson options are namespaces already, this option would get used as `-Drange-v3:range_v3_tests=true`. Don't include project names inside option names. Yes, I know you need it in cmake...

Reply:
i wanted to keep the variable name the same with the original repo (in upstream repo CMake configuration), but i agree having it just `tests` would have a better ergonomics.

Reply:
Still defaults to false?

Reply:
Umm, yes. Don't build the tests by default.

Reply:
Again, you need to make it build in CI. Previously it did so because it was enabled ~by default~ always.

See README.md.